docs/04_protocol/02_transactions_protocol.md

```diff
@@ -377,7 +377,7 @@ The `trx` field holds the transaction ID or the packed transaction itself. The a
 Deferred transactions are generated as a side effect of processing the blockchain, so their state is stored in the chain database, not within a block. Therefore, there is no need to explicitly include their contents in the transaction receipt. All in-sync nodes should be aware of the form of a deferred transaction as a matter of consensus. Deferred transactions issued by a smart contract have no role or effect on the `delayed` status field of the transaction receipt.
 
 [[caution | Deprecation Notice]]
-| Deferred transactions are deprecated as of EOSIO 2.0. For more details on their behavior, refer to the [Deferred Transactions](https://developers.eos.io/manuals/eosio.cdt/v1.7/best-practices/deferred_transactions) section on the `eosio.cdt` documentation.
+| Deferred transactions are deprecated as of Antelope 2.0. For more details on their behavior, refer to the [Deferred Transactions](https://developers.eos.io/manuals/eosio.cdt/v1.7/best-practices/deferred_transactions) section on the `eosio.cdt` documentation.
```

Comment:
Replacing to Antelope 2.0 is not applicable here. Created issue #17 to address these cases.